auth.ts

```
@@ -50,14 +50,29 @@ export const { handlers, signIn, signOut, auth } = NextAuth({
         }
       }
 
+      // Check if new session is exist
+      if (newSession && newSession?.user) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)